cpp/test/data/read_mesh.info

```diff
@@ -0,0 +1,49 @@
+type file
+
+mesh_file mesh_2d.ucd
+
+anode_collector_material_id   4
+anode_electrode_material_id   1
+separator_material_id         2
+cathode_electrode_material_id 3
+cathode_collector_material_id 5
+
+anode_collector_thickness    5.0e-4 ; [centimeter]
+anode_electrode_thickness   50.0e-4 ; [centimeter]
+separator_thickness         25.0e-4 ; [centimeter]
+cathode_electrode_thickness 50.0e-4 ; [centimeter]
+cathode_collector_thickness  5.0e-4 ; [centimeter]
+geometric_area              25.0e-2 ; [square centimeter]
+
+materials 4
+material_0 {    
+    name            anode
+    material_id         1
+}
+material_1 {
+    name        separator
+    material_id         2
+}
+material_2 {
+    name          cathode
+    material_id         3
+}
+material_3 {
+    name        collector
+    material_id       4,5
+}
+
+boundary_values {
+    anode_boundary_id   1
+    cathode_boundary_id 2
+
+    upper_boundary_id             3
+    lower_boundary_id             4
+    other_boundary_id             5
+
+    separator_material_id         3
+    anode_electrode_material_id   4
+    anode_collector_material_id   5
+    cathode_electrode_material_id 6
+    cathode_collector_material_id 7
+}
```

Comment:
This input file is the one used when you read the mesh from a file. You need them to know what's the anode, etc. right?